sdk/tests/js/tests/tex-image-and-sub-image-utils.js

```diff
@@ -163,7 +163,7 @@ var TexImageUtils = (function() {
     'in vec2 texCoord;',
     'out vec4 fragData;',
     'void main() {',
-    '    vec4 data = vec4(texture(tex, texCoord, 0).rgb 1.0);',
+    '    vec4 data = vec4(texture(tex, vec3(texCoord, 0.0)).rgb, 1.0);',
```

Comment:
This is incorrect.  For Sampler3dArray, it is texture(tex, xy, z(integer)), not texture(tex, xyz)